packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -314,4 +316,33 @@ export const selectIsDiscoveryAuthConfirmationRequired = (
     );
 };
 
+export const selectSupportedNetworks = (state: SuiteRootState) => {
+    const device = selectDevice(state);
+    const deviceModelInternal = device?.features?.internal_model;
+    const firmwareVersion = getFirmwareVersion(device);
+
+    return Object.entries(networks)
+        .map(([symbol, network]) => {
+            // @ts-expect-error TODO resolve why support is not there according to types
```

Comment:
`support` is only on ada/tada and networks is defined as const so the type is union of all network types, not merge... 

so something like 
`const support = 'support' in network ? network.support : undefined;` would work...